### ISSUE TITLE
Key <NSFaceIDUsageDescription> not added to info.plist

### ISSUE THREAD

**Issue by newfylox** [expo/expo#11262]
### Summary of Issue 

When using `CFBundleAllowMixedLocalizations` and adding `NSFaceIDUsageDescription` to all my locales, the build doesn't include that permission inside the `info.plist` file.

### Environment - output of `expo diagnostics` & the platform(s) you're targeting

  Expo CLI 4.0.8 environment info:
    System:
      OS: Linux 5.4 Ubuntu 20.04.1 LTS (Focal Fossa)
      Shell: 5.0.17 - /bin/bash
    Binaries:
      Node: 12.16.2 - ~/.nvm/versions/node/v12.16.2/bin/node
      Yarn: 1.22.5 - /usr/bin/yarn
      npm: 6.14.9 - ~/.nvm/versions/node/v12.16.2/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    SDKs:
      Android SDK:
        API Levels: 26, 27, 28, 29
        Build Tools: 27.0.3, 28.0.3
        System Images: android-28 | Intel x86 Atom_64, android-28 | Google Play Intel x86 Atom
    npmPackages:
      expo: ~39.0.3 => 39.0.3 
      react: 16.13.1 => 16.13.1 
      react-native: https://github.com/expo/react-native/archive/sdk-39.0.3.tar.gz => 0.63.2 
    npmGlobalPackages:
      expo-cli: 4.0.8
    Expo Workflow: managed

### What I did

I don't know if it's really possible to reproduce this issue. It could be a cache problem. I did clear it (`expo publish -c`) but it didn't help. I also deleted my `node_modules` folder and run `npm i` to install them again, didn't help. I did 2 builds and the two of them doesn't include the key \<NSFaceIDUsageDescription\> inside `info.plist` BUT do include the translations of `NSFaceIDUsageDescription` inside the IPA file

**Comment by cruzach** [comment]
@newfylox did you add `NSFaceIDUsageDescription ` to [`ios.infoplist`](https://docs.expo.io/versions/v40.0.0/config/app/#infoplist)?

**Comment by newfylox** [comment]
@cruzach at the time I wrote this issue no I didn't. But today I did it because I needed to submit the application on Apple Store.

My concern is that when using `CFBundleAllowMixedLocalizations`, I only put permissions inside my locale files and it works for them, but not for `NSFaceIDUsageDescription`. So it works for `NSCameraUsageDescription` and `NSPhotoLibraryUsageDescription` and they are not inside `ios.infoplist`, just in locale files.

I thought it would be the same for `NSFaceIDUsageDescription`. I just follow [this](https://docs.expo.io/distribution/app-stores/#localizing-your-ios-app)

**Comment by cruzach** [comment]
We include basic versions of (most, I think) of those strings automatically, but you need to customize them yourself if you want to make it through Apple's review process. There are docs on that [here](https://docs.expo.io/distribution/app-stores/#system-permissions-dialogs-on-ios) (same page you linked, just a little further up)

**Comment by newfylox** [comment]
Oh I see. Of course now I just saw that they talk about this [inside the note warning](https://docs.expo.io/versions/v39.0.0/sdk/local-authentication/#localauthenticationauthenticateasyncoptions) but Yesterday I was reading the top of the page in the [configuration section](https://docs.expo.io/versions/v39.0.0/sdk/local-authentication/#configuration) and I took for granted that if they talked about the Android permissions, they would have done the same about iOS permissions.

I guess I can close this issue than.
